GVFS/GVFS.Service/ProductUpgradeTimer.cs

```diff
@@ -98,5 +105,11 @@ private bool TryDownloadUpgrade(out string errorMessage)
                 }
             }
         }
+
+        private void EmptyDownloadDirectory()

```

Comment:
nit: I might suggest calling this `CleanupDownloadDirectory(...)`. `Empty` to me is a bit more ambiguous (is it used as a verb to "empty" a directory, ir is it checking if the directory is empty... ect...). 

As this is a private method and the calling code already has a product upgrade instance, I think we should re-use that instance.

Reply:
renamed to CleanupDownloadDirectory. Removed the private helper method.